definitions.py

```diff
@@ -0,0 +1,3 @@
+import os
```

Comment:
I'm assuming this is a stub which will be filled in with things later?

Reply:
The intention was to use this file to keep track of the project root via the `ROOT_DIR` variable. I thought I might be able to make use of it in other files in the project, specifically the preprocessing module. I didn't end up using it there, thought it might come in handy later.